tools/migrate_photo_from_odp.rb

```diff
@@ -0,0 +1,17 @@
+# Migrate all photos with restricted license (0) from the Open bucket to the Static bucket
+
+max_photo_id = LocalPhoto.last.id
+batch_size = 100000
+index = 0
+
+while index < max_photo_id
+	LocalPhoto.where( license: 0, file_prefix_id: 1 ).
```

Comment:
Ideally this ID wouldn't be hard coded. Might be better to use `FilePrefix.where( prefix: "https://inaturalist-open-data.s3.amazonaws.com/photos" ).first.id`. You could also set that to a variable before the loop and use the assigned variable here. We could also use `Shared::LicenseModule::C` instead of 0, but looks like we don't actually use that anywhere else in the codebase though maybe we should